lib/utils/testem-events.js

```diff
@@ -171,6 +171,23 @@ class TestemEvents {
     };
   }
 
+  /**
+   * Format module metadata map to json object
+   *
+   * @param {Object} moduleMetadata
+   */
+  _formatModuleMetadataMap(moduleMetadata) {
```

Comment:
it looks like this function is only adding `moduleName` field to each of the moduleMetadata object?
If so.. would adding that field in `_injectModuleMetadata` avoid the need for this function?
eg:
```
 _injectModuleMetadata(moduleName, total, passed, failed, runtime, failedTests) {
    this._moduleMetadata.set(
      moduleName,
      {
       moduleName
        total: total,
        passed: passed,
        failed: failed,
        runtime: runtime,
        failedTests: failedTests
      }
    );
}
```